.github/workflows/on-push-lint-charts.yml

```diff
@@ -0,0 +1,67 @@
+name: Lint and Test Charts
+
+on:
+  push:
+    paths:
+      - 'charts/**'
+      - '.github/**'
+env:
+  KUBE_SCORE_VERSION: 1.10.1
+  HELM_VERSION: 3.5.2
+  CHECK_VERSION_INCREMENT: false # avoid tests failing when multiple attempts to push chart changes are made
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v${{ env.HELM_VERSION }}
+
+      - name: Set up kube-score
+        run: |
+          wget https://github.com/zegl/kube-score/releases/download/v${{ env.KUBE_SCORE_VERSION }}/kube-score_${{ env.KUBE_SCORE_VERSION }}_linux_amd64 -O kube-score
+          chmod 755 kube-score
+
+      - name: Kube-score generated manifests
+        run: helm template  --values charts/.ci/values-kube-score.yaml charts/* | ./kube-score score -
```

Comment:
Can we call here `charts/.ci/scripts/local-kube-score.sh` instead? Otherwise we have to manage the exception list on multiple places.